### ISSUE TITLE
Unhandled exception while compiling

### ISSUE THREAD

**Issue by naveensrinivasan** [Fody/Costura#121]
I am getting an unhandled exception while compiling.

```

Error	1	Fody: An unhandled exception occurred:
Exception:
Could not load file or assembly 'Mono.Cecil, Version=0.9.6.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
StackTrace:
   at System.Signature.GetSignature(Void* pCorSig, Int32 cCorSig, RuntimeFieldHandleInternal fieldHandle, IRuntimeMethodInfo methodHandle, RuntimeType declaringType)
   at System.Reflection.RuntimeMethodInfo.FetchNonReturnParameters()
   at System.Reflection.RuntimeMethodInfo.GetParametersNoCopy()
   at System.Reflection.RuntimePropertyInfo.GetIndexParametersNoCopy()
   at System.Reflection.RuntimePropertyInfo.GetIndexParameters()
   at System.RuntimeType.GetPropertyCandidates(String name, BindingFlags bindingAttr, Type[] types, Boolean allowPrefixLookup)
   at System.RuntimeType.GetPropertyImpl(String name, BindingFlags bindingAttr, Binder binder, Type returnType, Type[] types, ParameterModifier[] modifiers)
   at System.Type.GetProperty(String name, BindingFlags bindingAttr, Binder binder, Type returnType, Type[] types, ParameterModifier[] modifiers)
   at PropertyDelegateBuilder.BuildPropertySetDelegate[T](Type type, String propertyName) in c:\ConsoleBuildAgent\work\ed448661dbb30d2e\FodyIsolated\DelegateBuilders\PropertyDelegateBuilder.cs:line 9
   at DelegateBuilder.BuildDelegateHolder(Type weaverType) in c:\ConsoleBuildAgent\work\ed448661dbb30d2e\FodyIsolated\DelegateBuilders\DelegateBuilder.cs:line 25
   at DelegateBuilder.GetDelegateHolderFromCache(Type weaverType) in c:\ConsoleBuildAgent\work\ed448661dbb30d2e\FodyIsolated\DelegateBuilders\DelegateBuilder.cs:line 16
   at InnerWeaver.InitialiseWeavers() in c:\ConsoleBuildAgent\work\ed448661dbb30d2e\FodyIsolated\InnerWeaver.cs:line 111
   at InnerWeaver.Execute() in c:\ConsoleBuildAgent\work\ed448661dbb30d2e\FodyIsolated\InnerWeaver.cs:line 61
Source:
mscorlib
TargetSite:
Void GetSignature(Void*, Int32, System.RuntimeFieldHandleInternal, System.IRuntimeMethodInfo, System.RuntimeType)
	WebAPIFileServer

```
Here is a link to the repo

https://github.com/naveensrinivasan/AspNetSelfHostFileServer/tree/fody-failing

**Comment by kavengagne** [comment]
Same problem exactly here.
Tried version 1.3.3 and 1.3.4, having the same problem.

**Comment by goldshtn** [comment]
Probably related to https://github.com/Fody/Costura/issues/118.

**Comment by SimonCropp** [comment]
try 1.3.5

**Comment by rosshays** [comment]
This issue was resolved for me by 1.3.5, thank you.

**Comment by naveensrinivasan** [comment]
Thanks!
